### DISCUSSION TITLE
Feature Request: Two Tables One Wheel

### DISCUSSION THREAD

**Discussion by topper-2k** [PinballY/PinballY-Addons-and-Examples#8] [Ideas]
I'm really liking the Two Tables One Wheel option. But I have a request.

It's weird to see "isAlternateVersion" in my list of categories. I mean, I know what it means, but it's odd and others using the cab don't understand it. Here's a sample of my categories.

Car & Driver
isAlternateVersion
Marvel
Movie Monsters
Music
Star Trek
Star Wars
...

isAlternateVersion sticks out like a sore thumb. A more intuitive option would be "Alternate Versions," So my modified list would be:

Alternate Versions
Car & Driver
Marvel
Movie Monsters
Music
Star Trek
Star Wars
...

Any chance we could make that change? Or maybe allow the user to define their own alt category label?

**Comment by stephenhouser** [comment]
Good idea. Don't think it would be much code to implement either. 

The approach I'm thinking of would require you edit the config file by hand though. Would that be reasonable?

**Comment by topper-2k** [comment]
That would work perfectly for me.

**Reply by stephenhouser** [reply]
Apologies for the delay, have not been able to get at my table for a bit. A quick and dirty solution would be to 

1. change `isAlternateVersion` on line 31 of `two-tables-one-wheel.js` to whatever label you want to use. In your case `Alternate Versions`. Change:
```
    return game.categories.indexOf("isAlternateVersion") >= 0;
```
to
```
    return game.categories.indexOf("Alternate Versions") >= 0;
```

2. Change your categories to the same thing using PinballY as normal.

NOTE: I've not been able to test this but have 90% confidence it will work.

**Comment by stephenhouser** [comment]
I just pushed a change that will enable the ability to change the category used. It also changes the text of the menu item "Show Alternate Versions" as well.

NOTE: This also changes the default from `isAlternateVersion` to `Alternate Version` for the category and will break older installs.

**Reply by topper-2k** [reply]
Just got to try this out. Sorry for the delay, some other things got in the way of virtual pinball for a bit.

This works great. Many thanks!!

**Comment by stephenhouser** [comment]
Please put future feature requests as issues rather than discussions. Easier to "close" them that way.
